src/dbus/client/thread_api_dbus.hpp

```diff
@@ -369,6 +369,18 @@ class ThreadApiDBus
      */
     ClientError SetRadioRegion(const std::string &aRadioRegion);
 
+    /**
+     * This method sets the NAT64 switch.
+     *
+     * @param[in] aEnable  A boolean to enable/disable the NAT64.
+     *
+     * @retval ERROR_NONE  Successfully performed the dbus function call
+     * @retval ERROR_DBUS  dbus encode/decode error
+     * @retval ...         OpenThread defined error value otherwise
+     *
+     */
+    ClientError SetNat64Enabled(bool aEnabled);
```

Comment:
Except for the test, we don't expect other caller of this API. Is this correct?

My understanding is that setting nat64 from feature flags list will be handled in `ControllerOpenThread::ApplyFeatureFlagList`.

Reply:
This is expected.

`ApplyFeatureFlagList` should call `otNat64SetEnabled` instead of this.